contracts/counter/examples/cli.rs

```diff
@@ -0,0 +1,16 @@
+use cosmwasm_std::Empty;
+use counter_contract::CounterContract;
+use cw_orch::{anyhow, prelude::Daemon, tokio::runtime::Runtime};
+use cw_orch_contract_cli::{ContractCli, DaemonFromCli};
+
+pub fn main() -> anyhow::Result<()> {
```

Comment:
What does this example show? When could someone use this. 
Add a description as a doc-comment to answer those questions.